src/SendHeaders.php

```diff
@@ -60,7 +60,9 @@ public function setCacheHeader(): void
 			}
 
 			if ($this->pageProperties['isFrontPage']) {
-				$this->frontPageCacheAge = get_field('cache_control_plugin_front_page_cache', 'option');
+				if (is_string(get_field('cache_control_plugin_front_page_cache', 'option'))) {
```

Comment:
We should probably also encapsulate this check when we come back to this code later